style.css

```diff
@@ -0,0 +1,175 @@
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Ubuntu", sans-serif;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 1.5;
+  color: lightskyblue;
+}
+
+h1 {
+  font-size: 55px;
+}
+
+h2 {
+  font-size: 45px;
+}
+
+h1,
+h2 {
+  font-family: "Ubuntu", sans-serif;
+  font-weight: 700;
+  text-align: center;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: palegoldenrod;
+}
+
+img {
+  width: 100%;
+  display: block;
+}
+
+.nav_menu {
+  width: 100%;
+  top: 0;
+  left: 0;
+  position: fixed;
+  display: flex;
+  justify-content: flex-end;
+  background-color: maroon;
+  z-index: 20;
+}
+
+.nav-items {
+  display: flex;
+  margin-right: 15px;
+}
+
+.nav-items a {
+  display: block;
+  font-size: 20px;
+  padding: 20px;
+}
+
+.nav-items a:hover {
+  background: indigo;
+}
+
+.intro-sect {
```

Comment:
Considering that the class "intro-sect" is applied to only one section, please select this section by its "id" attribute #welcome-section